### PR TITLE
Remove mentions of exclude_paths

### PR DESCRIPTION
**Pull request by wfleming**
Exclude paths have been deprecated for some time. To avoid confusion, I
think it's time we remove them completely.

This justifies a version bump to `0.2.0` as well, I think.

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -41,8 +36,6 @@ These are covered in detail below.
 
 `include_paths` may include paths to files that are irrelevant to the analysis (i.e., `.gitignore` if the engine only analyzes JavaScript). Engines are responsible for filtering out irrelevant files.
```
Comment (jpignata) [inline SPEC.md]: Not related to anything you changed, but I find this i.e. aside confusing.

**Comment by jpignata** [comment]
LGTM cc: @mrb

**Comment by mrb** [comment]
@wfleming I'm +1 on the change and the bump to `0.2.0`, but I think we need to read through the whole SPEC again to make sure it's coherent with this change - did you do that or should I do a run through as well?

**Comment by wfleming** [comment]
@mrb I was focused mainly on the flow between the **Input** & **Which files to analyze** sections, but I did do a quick read through of the rest of the doc as well. I didn't notice anything that conflicted with this,

**Pull request merged**